ang/civicase/CaseListTable.html

```diff
@@ -44,61 +44,14 @@
   </thead>
   <tbody>
   <tr ng-repeat="item in cases">
-    <td ng-click="viewCase(item.id, $event)" ng-class="{'case-selected-for-view': viewingCase === item.id}" class="civicase__case-list-column--fixed">
-      <div ng-if="!bulkAllowed" class="lockDiv">
-        <div class="fa fa-lock"></div>
-      </div>
-      <div class="case-list-checkbox" ng-if="!viewingCase && selectedCases && bulkAllowed">
-        <input type="checkbox" ng-model="item.selected" />
-        <label></label>
-      </div>
-      <div class="case-list-info">
-        <h4>{{ item.client[0].display_name }}</h4>
-        <div class="btn-group btn-group-sm" ng-if="item.client.length > 1">
-          <button type="button" class="btn btn-info dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" title="{{ ts('Additional clients') }}">
-            +{{ item.client.length - 1 }}
-          </button>
-          <ul class="dropdown-menu" >
-            <li ng-repeat="(index, client) in item.client" ng-if="index">
-              <span>{{ client.display_name }}</span>
-            </li>
-          </ul>
-        </div>
-        <div ng-if="item.client[0].email">
-          <strong>{{ ts('Email:') }}</strong>
-          {{ item.client[0].email }}
-        </div>
-        <div ng-if="item.client[0].phone">
-          <strong>{{ ts('Phone:') }}</strong>
-          {{ item.client[0].phone }}
-        </div>
-        <div>
-          <strong>{{ ts('Case ID:') }}</strong>
-          {{ item.id }} &nbsp;
-          <span class="civicase-tags">
-            <span
-              class="label label-default"
-              title="{{ tag['tag_id.description'] }}"
-              ng-repeat="tag in item.tag_id || []"
-              ng-class="{'text-dark': CRM.utils.colorContrast(tag['tag_id.color'] || '#cccccc') === 'black'}"
-              style="background-color: {{ tag['tag_id.color'] }};"
-            >{{ tag['tag_id.name'] }}</span>
-          </span>
-        </div>
-        <div class="case-activity-summary-icons" ng-if="bulkAllowed">
-          <a class=""
-             ng-repeat="category in activityCategories"
-             ng-href="#{{ activityFeedUrl(item.id, category.name, 'incomplete') }}"
-             ng-class="{'item-overdue': item.activity_summary[category.name][0].is_overdue}"
-             ng-if="item.category_count.incomplete[category.name]"
-             title="{{ ts('%1 upcoming %2', {1: item.category_count.incomplete[category.name], 2: category.label}) }}"
-          >
-            <i class="fa {{ category.icon }}"></i>
-            <span class="badge noti-badge">{{ item.category_count.incomplete[category.name] }}</span>
-          </a>
-        </div>
-      </div>
+    <td ng-click="viewCase(item.id, $event)" ng-class="{'civicase__case-card--active': viewingCase === item.id}" class="" ng-include="'~/civicase/CaseCard.html'">
```

Comment:
`ng-class="{'civicase__case-card--active': viewingCase === item.id}"` should be moved to CaseCard.html. As applying the active class is the job the directive controller, not the parent controller.